src/main/webapp/app/entities/student-with-team.model.ts

```diff
@@ -0,0 +1,7 @@
+export class StudentWithTeam {
+    public name: string;
```

Comment:
```suggestion
    public firstName?: string;
```

Reply:
I think this can be optional as we identify students either by username or registration number